### PR TITLE
adding zypper support

### PR DESCRIPTION
**Pull request by colbec**

### CODE REVIEW

**Comment by nalimilan** [comment]
The failure is due to the fact that no released BinDeps version exports `Zypper` yet. But the fix looks good to me it if works for you -- except that you'll have to require BinDeps 0.3.21 once it's released.

**Comment by colbec** [comment]
Got it, thankyou. I will hold fire on this, nothing urgent.

File: REQUIRE [hunk 1]
```diff
@@ -1,7 +1,7 @@
 julia 0.3
 Compat 0.3.5
 Colors
-BinDeps
+BinDeps 0.3.20
```
Comment (tkelman) [inline REQUIRE]: This can be rebased now that 0.3.21 was tagged.

**Comment by timholy** [comment]
Thanks!

**Pull request merged**